apps/judicial-system/backend/src/app/modules/subpoena/test/internalSubpoenaController/deliverSubpoenaToCourt.spec.ts

```
@@ -0,0 +1,111 @@
+
+      await internalSubpoenaController
+        .deliverSubpoenaToCourt(
+          caseId,
+          defendantId,
+          subpoenaId,
+          theCase,
+          defendant,
+          subpoena,
+          dto,
+        )
+        .then((result) => (then.result = result))
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)